orchestrator/test_runner/src/transaction_stress_test.rs

```diff
@@ -88,9 +88,8 @@ pub async fn transaction_stress_test(
         );
     }
 
-    let start = Instant::now();
     let mut good = true;
-    while Instant::now() - start < TOTAL_TIMEOUT {
+    match tokio::time::timeout(TOTAL_TIMEOUT, async {
         good = true;
```

Comment:
Since we're not using a `while` loop any more we don't have to set `good = true` at the top, it was already defined as true above.